examples/forum/forumproto.nim

```diff
@@ -3,18 +3,16 @@ import ormin, ormin/serverws, json
 
 importModel(DbBackend.sqlite, "forum_model")
 
+static:
+  dbtypetables.add(dbInet, "string")
+  
 var db {.global.} = open("stuff", "", "", "")
 
 protocol "forumclient.nim":
   common:
     when defined(js):
-      type kstring = cstring
-    else:
-      type kstring = string
-    type
-      inet = kstring
-      varchar = kstring
-      timestamp = kstring
+      type string = cstring
```

Comment:
This is too bad code for an "example".

Reply:
New solution:
```
protocol "chatclient.nim":
  common:
    typemap:
      kstring = string
    when defined(js):
      type kstring = cstring
    else:
      type kstring = string
```
Declare kstring equivalent to replace string using typemap in common, the chat sample generate follow code:
```
when defined(js):
  type
    kstring = cstring
else:
  type
    kstring = string
type
  TextMessage* = ref object
    content*: kstring
    creation*: kstring
    author*: int
    name*: kstring
```
This will not bother users, will it?